windows/regf.ksy

```diff
@@ -248,8 +252,8 @@ types:
           - id: value_name
             size: value_name_size
             type: str
-            encoding: ascii
-            if: "flags == vk_flags::value_comp_name"
+            encoding: iso-8859-1
+            if: 'flags == vk_flags::value_comp_name'
```

Comment:
Is there any deep reason to replace double quotes with single ones in this statement, taking into account that the rest file uses double quotes?